src/main/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource.java

```diff
@@ -267,7 +267,8 @@ public OffenderDetailSummary getOffenderDetails(
         final @RequestParam(value = "complied", required = false) Optional<Boolean> complied,
         final @RequestParam(value = "nationalStandard", required = false) Optional<Boolean> nationalStandard,
         final @RequestParam(value = "outcome", required = false) Optional<Boolean> outcome,
-        final @ApiParam(name = "include", value = "Contacts to include. Can be a contact type code, prefixed with 'type_' or appointments", example = "type_ccmp", allowMultiple = true) @RequestParam(value = "include", required = false) Optional<List<String>> include) {
+        final @ApiParam(name = "include", value = "Contacts to include. Can be a contact type code, prefixed with 'type_' or appointments", example = "type_ccmp", allowMultiple = true) @RequestParam(value = "include", required = false) Optional<List<String>> include,
+        final @RequestParam(value = "rarActivity", required = false) Optional<Boolean> rarActivity) {
```

Comment:
Looks like you are missing Swagger for this, which I think is important in this case

Reply:
Done